lib/src/inbox_message_view.dart

```diff
@@ -63,4 +71,12 @@ class _InboxMessageViewState extends State<InboxMessageView> {
 
     return new Text('$defaultTargetPlatform is not yet supported by this plugin');
   }
+
+  dispose() {
+    super.dispose();
+
+    if (widget.onClose != null) {
+      widget.onClose();
```

Comment:
This is not that the widget closed, but if `UAirship.close()` is called from within the message view.